lib/board.rb

```diff
@@ -2,15 +2,16 @@ class Board
   attr_reader :visual, :last_piece_played
 
   def initialize
-    @visual = {
-      "A" => %w[. . . . . .],
-      "B" => %w[. . . . . .],
-      "C" => %w[. . . . . .],
-      "D" => %w[. . . . . .],
-      "E" => %w[. . . . . .],
-      "F" => %w[. . . . . .],
-      "G" => %w[. . . . . .]
-    }
+    @visual = Hash.new { |hash, key| hash[key] = Array.new(6, '.') }
+    # @visual = {
+    #   "A" => %w[. . . . . .],
+    #   "B" => %w[. . . . . .],
+    #   "C" => %w[. . . . . .],
+    #   "D" => %w[. . . . . .],
+    #   "E" => %w[. . . . . .],
+    #   "F" => %w[. . . . . .],
+    #   "G" => %w[. . . . . .]
+    # }
```

Comment:
Like the cool new way to generate the board. Will need to clean up commented out old board